src/content/app/genome-browser/components/track-panel/components/track-panel-list/track-panel-items/TrackPanelGene.tsx

```diff
@@ -144,12 +144,21 @@ const TrackPanelGene = (props: TrackPanelGeneProps) => {
     );
   };
 
+  const getVisibilityIconHelpText = (status: Status) => {
+    if (status === Status.SELECTED) {
+      return 'Hide all transcripts';
+    }
+
+    return 'Show all transcripts';
+  };
+
   return (
     <>
       <GroupTrackPanelItemLayout
         isCollapsed={isCollapsed}
         visibilityStatus={geneVisibilityStatus}
         onChangeVisibility={onGeneVisibilityChange}
+        getVisibilityIconHelpText={getVisibilityIconHelpText}
```

Comment:
I'm confused: doesn't `TrackPanelGene` at this point know what its visibility status is? Why wouldn't it just tell `GroupTrackPanelItemLayout` as a string what the help text ought to be? Why pass the function?

Reply:
I've updated it to pass the help text as  a prop instead of a function.